crates/uv-client/Cargo.toml

```diff
@@ -47,5 +48,8 @@ urlencoding = { workspace = true }
 
 [dev-dependencies]
 anyhow = { workspace = true }
+http-body-util = { version = "0.1.0" }
+hyper = { version = "1.2.0", features = ["server", "http1"] }
```

Comment:
Can we write the tests to use `hyper v0.14.28`? `reqwest` isn't on v1.0, and this adds a lot of dependencies which hurts compile times etc.

Reply:
Good point, will do.

Reply:
Done in 35eee6d53bdaecea9777635ac74421af42bf9012 I was surprised about how different some things are in 1.x vs 0.14.x.

Reply:
Thank you!